app/views/main/question_list_helper.html

```diff
@@ -0,0 +1,44 @@
+<header>
+  <a href="index.html" id="back" data-icon="back-light" data-action="back">
+    <span class="visuallyhidden">{{ _('Close') }}</span>
+  </a>
+  <menu type="toolbar">
+    <a href="profile.html" id="settings" class="settings-icon" data-icon="settings" data-modal="true">
+      <span class="visuallyhidden">{{ _('Settings') }}</span>
+    </a>
+  </menu>
+  <h1 id="main_heading">{{ _('Answer a Question') }}</h1>
+</header>
+<main class="fit scroll">
+  <section class="tabbed-section" role="region">
+    <div id="sign-in-link" class="ql-signIn hide">
+      <p class="ql-signIn-label">{{ _('Sign in to answer questions from Firefox OS users.') }}</p>
+      <a href="authentication.html" data-modal="true" class="bb-button recommend">{{ _('Create account or sign in') }}</a>
+    </div>
+    <ul id="new-active-tabs" role="tablist" class="bb-tablist" data-type="filter">
+      <li id="new" role="presentation">
+        <a id="tab1" href="#panel_new" role="tab" aria-controls="panel_new" aria-selected="true">{{ _('New') }}</a>
+      </li>
+      <li id="active" role="presentation">
+        <a id="tab2" href="#panel_active" role="tab" aria-controls="panel_active" aria-selected="false">{{ _('My list') }}</a>
+      </li>
+    </ul>
+
+    <div id="panel_new" class="bb-tabpanel questionList" role="tabpanel" aria-labelledby="tab1">
+      <section id="myquestions" class="questions js-question-container all" data-all="true"></section>
+      <a href="#" role="button" id="load-more-my-questions" data-icon="download" class="ql-loadMore hide">{{ _('Load more') }}</a>
+      <div id="refresh-toolbar" class="ql-bottomToolbar hide">
+        <a href="#" role="button" id="refresh-my-questions" class="ql-refresh" data-icon="sync"></a>
+              <span>{{ _('Last sync:') }}
+                <time id="last-refreshed-time">{{ _('x hours ago') }}</time>
```

Comment:
I think this is going to be kind of goofy and not-localization friendly. Also, not RTL friendly.